exercises/exercise-3.js

```diff
@@ -6,11 +6,34 @@
 
   Considerations: Values can be nested and datypes Number, String or Object.
 */
+const obj = {
+  a: "Hello",
+  b: {
+    c: "Hey",
+    d: 23,
+    f: {
+      g: "A string",
+      h: {
+        i: "Another string",
+      },
+    },
+  },
+  e: 34,
+};
 
-function deepStringSum() {
-  // TODO: implement
+let counter = 0;
+function deepStringSum(obj) {
+  for (let key in obj) {
+    if (typeof obj[key] == "object" && obj[key] !== null) {
```

Comment:
Be careful when you use `==`, in some cases you could receive unexpected results, in major of cases `===` is better because you're forcing data.